src/main/java/org/apache/commons/io/input/BoundedInputStream.java

```diff
@@ -427,13 +421,19 @@ public long getMaxLength() {
     }
 
     /**
-     * Gets how many bytes remain to read.
+     * Returns the bytes remaining before the configured read limit is reached.
```

Comment:
See above about documenting getter methods.

Reply:
Fixed in https://github.com/apache/commons-io/pull/779/commits/c831b562f656559e246da43197cf8fe7d5751f45